coopr-docs/docs/source/installation/upgrade.rst

```diff
@@ -0,0 +1,71 @@
+..
+   Copyright © 2015 Cask Data, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ 
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+.. index::
+   single: Coopr Upgrade
+
+==================
+Coopr Upgrade
+==================
+
+.. highlight:: console
+
+This is an overview of the upgrade path from Coopr 0.9.8 to Coopr 0.9.9.
+
+Upgrade Steps
+-------------
+As with any system upgrades, please back up all critical data, prior to running any of the steps below.
+We are assuming you have access to all (current 0.9.8) Coopr servers and are logged in as root on those.
+
+1. Stop all processes
+
+  .. parsed-literal::
+   $ sudo /etc/init.d/coopr-provisioner stop
+   $ sudo /etc/init.d/coopr-ui stop
+   $ sudo /etc/init.d/coopr-server stop
+
+2. Back up your database
+
+  * It is always recommended to back up databases. For MySQL, you can use mysqldump to do that.
```

Comment:
Don't talk about `mysqldump`.

"It is recommended to backup your database. Consult your database vendor's documentation for instructions."